static/dream_web/index.js

```diff
@@ -59,7 +59,8 @@ async function generateSubmit(form) {
 
     // Convert file data to base64
     let formData = Object.fromEntries(new FormData(form));
-    formData.initimg = formData.initimg.name !== '' ? await toBase64(formData.initimg) : null;
+    formData.initimg_name  = formData.initimg.name
+    formData.initimg       = formData.initimg.name !== '' ? await toBase64(formData.initimg) : null;
```

Comment:
Just a style note: absolutely no one writes JS this way, with aligned RHSes. It's your project so you can do what you want, but this is reads very, very strangely to anyone with a JS background.

Reply:
I guess I have an idiosyncratic style dating from my FORTRAN77 days. :-(

Do people write python with aligned RHSes, or am I driving people nuts when I do that too?

Reply:
I write a lot less Python than I do JS, so I can't say for sure how common this style is there, but I haven't seen it personally. I do know that the [official style guide](https://peps.python.org/pep-0008/#other-recommendations) as well as the fairly-popular [Google style guide](https://google.github.io/styleguide/pyguide.html#36-whitespace) officially discourage aligned RHSes. Quoth Google:

> Don’t use spaces to vertically align tokens on consecutive lines, since it becomes a maintenance burden (applies to :, #, =, etc.)